Source/Core/Core/AchievementManager.cpp

```diff
@@ -863,7 +863,10 @@ void AchievementManager::ChangeMediaCallback(int result, const char* error_messa
 {
   AchievementManager::GetInstance().m_loading_volume.reset(nullptr);
   if (result == RC_OK)
```

Comment:
Is this the only "good" result?  Should `RC_HARDCORE_DISABLED` be an error?

Reply:
I was answered on discord.  This can be resolved (still wish user could resolve issues..)

Reply:
Actually, no, you're right - I should be setting the function there too.